src/createXcmTypes/types.ts

```diff
@@ -41,10 +41,15 @@ export interface ICreateXcmType {
 		xcmVersion: number,
 		specName: string,
 		assets: string[],
-		opts: CreateAssetsOpts
-	) => VersionedMultiAssets;
+		opts: CreateAssetsOpts,
+		transferForeignAssets?: boolean | undefined
```

Comment:
As mentioned earlier it would be nice to see `transferForeignAssets` to `opts`, and rename it to `isForeignAssetsTransfer`